docs/latest/user/udp-routing.md

```diff
@@ -0,0 +1,196 @@
+# UDP Routing
+
+The [UDPRoute](https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io/v1alpha2.UDPRoute/)
+resource allows users to configure UDP routing by matching UDP traffic and forwarding it to
+Kubernetes backends. This guide will use a CoredDNS example to walk you through the steps required to configure
+ UDPRoute on Envoy Gateway.
+
+## Prerequisites
+
+- A Kubernetes cluster with `kubectl` context configured for the cluster.
+
+__Note:__ Envoy Gateway is tested against Kubernetes v1.24.
+
+## Installation
+
+Follow the steps from the [Quickstart Guide](quickstart.md) to install Envoy Gateway.
+
+Install a CoreDNS in the Kubernetes cluster as the example backend. The installed CoreDNS is listening on
+ the UDP port 53 for DNS lookups.
+
+```shell
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
```

Comment:
Since the example backend is fairly large, let's follow the approach of the HTTP Routing user guide and `kubectl apply -f https://raw.githubusercontent.com/envoyproxy/gateway/latest/examples/kubernetes/udp-routing.yaml`. The [docs target](https://github.com/envoyproxy/gateway/blob/main/tools/make/docs.mk#L45-L55) will handle updating the release tag.